AzureCommunicationUI/AzureCommunicationUI/Presentation/FluentUI/Wrapper/CompositeParticipantsListCell.swift

```diff
@@ -7,15 +7,26 @@ import Foundation
 import FluentUI
 
 class CompositeParticipantsListCell: TableViewCell {
-    func setup(displayName: String, isMuted: Bool, accessoryType: TableViewCellAccessoryType = .none) {
+
+    /// Set up the participant list item  in the participant list
+    /// - Parameters:
+    ///   - displayName: The display name shown on the participant list cell
+    ///   - isMuted: tells whether this participant is muted
+    ///   - partipantHasEmptyName: tells whether this participant had an empty name when joined the call
+    ///   - accessoryType: accessory type
+    func setup(displayName: String,
+               isMuted: Bool,
+               partipantHasEmptyName: Bool,
```

Comment:
2 things. 
Spelling here:
```suggestion
               participantHasEmptyName: Bool,
```
And if we are now passing in a boolean to represent this I don't think we need the 
`let isNameEmpty = displayName.trimmingCharacters(in: .whitespaces).isEmpty` so we should remove that.